docs/src/modules/java/pages/call-another-service.adoc

```diff
@@ -0,0 +1,57 @@
+= Calling other services
+
+== Akka Serverless services
+
+In some cases it is useful to call a component in another service, for example interacting with multiple entities or actions, aggregating their return values or performing a multi-step workflow. xref:actions-as-controller.adoc[Forwards] and xref:side-effects.adoc[Side Effects] allow us to trigger other services but does not make it possible to compose and transform the results of calling them.
+
+Calling other Akka Serverless services in the same project from an Action is done by invoking them over gRPC much like how an external client would. The service is however identified only by the name it has been deployed as, Akka Serverless takes care of routing requests to the service and keeping the data safe by encrypting the connection for us.
+
+In this sample we will make an action that does two sequential calls to the xref:value-entity.adoc[Value Entity Counter] service, deployed with the service name "counter".
+
+We start by adding the public API of the counter to the `src/main/proto` directory of our project.
+
+Since the proto file of an Akka Serverless service contains annotations that cause the Java SDK code generation to generate services, and we only want to consume the service, we need to start by removing the annotations.
+
+Copy the API definition proto file from the other service into the `proto` directory but strip all `akkaserverless.service` option blocks as well as all other Akka Serverless annotations and the `import "akkaserverless/annotations.proto"` from it.
```

Comment:
is it clear that "strip all" means "remove all"? otherwise we could say less fancy "remove all"